### DISCUSSION TITLE
Rights to use PocketBase in production?

### DISCUSSION THREAD

**Discussion by jannismilz** [pocketbase/pocketbase#1264] [Q&A] [answered]
Do I have the right to use PocketBase in production for my project and maybe even use it commercially?

- Any attribution needed?
- Anything I need to pay attention to?

**Comment by ganigeorgiev** [comment, marked as answer]
Yes you are free to use it, it is [MIT licensed](https://github.com/pocketbase/pocketbase/blob/master/LICENSE.md) and no attribution is needed. You can even offer it as a paid service if you want to.

The only thing that you have to keep in mind is that PocketBase is still in very active development stage and full backward compatibility is not guaranteed before reaching v1.0.0 (there is no hard ETA for v1, but I try to follow the [public roadmap](https://github.com/orgs/pocketbase/projects/2)).

**Reply by jannismilz** [reply]
Awesome. Thanks alot!
